modules/dxp/apps/portal-search-tuning/portal-search-tuning-blueprints-admin-web/test/stories/index.js

```diff
@@ -134,6 +136,7 @@ storiesOf('Pages|BlueprintForm', module)
 			context={CONTEXT}
 			props={{
 				...BLUEPRINT_FORM_PROPS,
+				blueprintId: '0',
```

Comment:
Is this needed?

Reply:
That case seems to give an error without that line, but I don't think it matters too much since this case gets removed in the next commit